### ISSUE TITLE
[Spike] Cloudflare CDN 

### ISSUE THREAD

**Issue by safaci2000** [OSAlt/sysadmin#4]
Look into Cloudfare CDN and see what's involved.

benefits.
requirements.
cost if any.

**Comment by safaci2000** [comment]
@drahamim  can you elaborate on the advantages of this? cost ? and why we care? I'm assuming it hides the public IP anything else we should know?

**Comment by mjeveritt** [comment]
I don't think we have enough raw content to be bothered by Content-Delivery in that way .. DDoS protection may be a concern, but I've never seen a problem much worse than script kiddies attempting breakage on Wordpress sites with known vulnerabilities....

Personally, I'm nervous about the terms 'Personal' and 'Evaluation' wrt to the 'Free' tiers of Cloudflare .. it would be fine if used for a Personal site, but if GB takes off .. it's gonna quickly attract attention from sysadmins (external to ourselves!) to require funding or shutdown .. that's not a scenario I particularly wanna cross .. being blackmailed ;)

Just my usual $2 .. apologies in advance! :D

**Comment by drahamim** [comment]
I wasn't really aware of the TOS for free tier since I just use it for my personal site. We could try it out and see what the stats are for the site. Then if we want and it seems needed we could move to the next tier up which isnt that costly. 
Alternatively AWS does over some low level DDOS protection as part of EC2.
